### PR TITLE
Schmiedeberg Tweaks And Fixes

### PR DESCRIPTION
**Pull request by goober3**
## About The Pull Request

Added a shuttle expansion permit to chief pharmacist's locker
Added more seeds to the seed crate (replica pods, aloe, corn)
There is no longer a Random Defib Mount in the chief pharmacist's locker
The chemical press is now Rotated Correctly 

## Why It's Good For The Game

tweaks r good. 

## Changelog

:cl:
tweak: The Schmiedeberg now has shuttle expansion permit, extra seeds have been added to the seed crate.
/:cl:

### CODE REVIEW

File: _maps/shuttles/shiptest/independent_schmiedeberg.dmm [hunk 1]
```diff
@@ -657,11 +657,13 @@
 	dir = 4
 	},
 /obj/structure/closet/crate/hydroponics,
-/obj/item/seeds/apple,
 /obj/item/seeds/ambrosia,
 /obj/item/seeds/glowshroom,
 /obj/item/seeds/random,
+/obj/item/seeds/replicapod,
```
Comment (Erikafox) [inline _maps/shuttles/shiptest/independent_schmiedeberg.dmm]: no roundstart replica pods

File: _maps/shuttles/shiptest/independent_schmiedeberg.dmm [hunk 2]
```diff
@@ -2379,7 +2381,7 @@
 /obj/item/assembly/flash/handheld,
 /obj/item/door_remote/chief_medical_officer,
 /obj/item/pet_carrier,
-/obj/item/wallframe/defib_mount,
```
Comment (Erikafox) [inline _maps/shuttles/shiptest/independent_schmiedeberg.dmm:2382]: Is there a defib on the ship that this was intended for or was it truly random

**Review by Erikafox — CHANGES_REQUESTED** [review]

File: _maps/shuttles/shiptest/independent_schmiedeberg.dmm (hunk 1 above)
Comment (goober3) [reply _maps/shuttles/shiptest/independent_schmiedeberg.dmm]: nineteen eighty four

File: _maps/shuttles/shiptest/independent_schmiedeberg.dmm (hunk 2 above)
Comment (goober3) [reply _maps/shuttles/shiptest/independent_schmiedeberg.dmm:2382]: yeah, there's not a single defib on this ship. but there's a defib mount in the locker. odd!

File: _maps/shuttles/shiptest/independent_schmiedeberg.dmm (hunk 1 above)
Comment (Erikafox) [reply _maps/shuttles/shiptest/independent_schmiedeberg.dmm]: give them cabbages to mutate if you *have* to

Comment (goober3) [reply _maps/shuttles/shiptest/independent_schmiedeberg.dmm]: yeah, I figured replica pods would be a neat method of reviving people for a botanical medical ship, without a defib.

**Review by Erikafox — APPROVED** [review]

**Pull request merged**